operators/aqua/6.5.5/manifests/aqua-operator.v6.5.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,574 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: aqua-operator.v6.5.5
+  namespace: placeholder
+  annotations:
+    # Setting olm.maxOpenShiftVersion automatically
+    # This property was added via an automatic process since it was possible to identify that this distribution uses API(s),
+    # which will be removed in the k8s version 1.22 and OpenShift version OCP 4.9. Then, it will prevent OCP users to
+    # upgrade their cluster to 4.9 before they have installed in their current clusters a version of your operator that
+    # is compatible with it. Please, ensure that your project is no longer using these API(s) and that you start to
+    # distribute solutions which is compatible with Openshift 4.9.
+    # For further information, check the README of this repository.
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.10"}]'
```

Comment:
@yossig-aquasec , 

Same here. Could you please fix it prior to the PR getting merged?

c/c @J0zi @mvalarh